app/conf.js

```diff
@@ -42,7 +42,23 @@ const conf = {
   maintenance: {
     flag: process.env.MAINTENANCE_FLAG,
     key: process.env.MAINTENANCE_KEY
-  }
+  },
+  kms: {
+    server: {
+      serviceUrl: process.env.KMS_SERVER_SERVICE_URL || 'https://us-south.kms.cloud.ibm.com',
+      iamAuthUrl: process.env.KMS_SERVER_IAM_AUTH_URL || 'https://iam.cloud.ibm.com',
```

Comment:
Can we avoid adding IBM cloud defaults, especially `us-south` specific defaults?  If possible, avoid needing a default at all -- if the value isn't set, code shouldn't be attempting to use KMS.
```suggestion
      serviceUrl: process.env.KMS_SERVER_SERVICE_URL,
      iamAuthUrl: process.env.KMS_SERVER_IAM_AUTH_URL,
```